### ISSUE TITLE
[iOS] App crashes on start-up due to resource path changes

### ISSUE THREAD

**Issue by rh101** [axmolengine/axmol#1087]
- axmol version: latest
- devices test on: iOS simulator and device
- developing environments
   - NDK version: r19c
   - Xcode version: 12.4
   - Visual Studio: 
      - VS version: 2019 (16.11), 2022 (17.4)
      - MSVC version: 1929, 1934
      - Windows SDK version: 10.0.22621.0
   - cmake version: 
Steps to Reproduce:

1. create a new project from the Axmol cpp template 
2. build and run the project

It will crash with the following output:

```
xmol: fullPathForFilename: No file found at CloseNormal.png. Possible missing file.
axmol: fullPathForFilename: No file found at CloseSelected.png. Possible missing file.
Error while loading: 'CloseNormal.png' and 'CloseSelected.png'
Depending on how you compiled you might have to add 'Resources/' in front of filenames in HelloWorldScene.cpp
axmol: fullPathForFilename: No file found at fonts/Marker Felt.ttf. Possible missing file.
Error while loading: 'fonts/Marker Felt.ttf'
Depending on how you compiled you might have to add 'Resources/' in front of filenames in HelloWorldScene.cpp
axmol: fullPathForFilename: No file found at HelloWorld.png. Possible missing file.
Error while loading: 'HelloWorld.png'
Depending on how you compiled you might have to add 'Resources/' in front of filenames in HelloWorldScene.cpp
2023-03-01 17:51:38.692532+1100 HelloCpp[4211:37333] fopen failed for data file: errno = 2 (No such file or directory)
2023-03-01 17:51:38.692783+1100 HelloCpp[4211:37333] Errors found! Invalidating cache...
axmol: QuadCommand: resizing index size from [-1] to [2560]
2023-03-01 17:51:39.171361+1100 HelloCpp[4211:37333] fopen failed for data file: errno = 2 (No such file or directory)
2023-03-01 17:51:39.171471+1100 HelloCpp[4211:37333] Errors found! Invalidating cache...
```

**Comment by rh101** [comment]
Does iOS look into a "Resources" folder by default? I can't find any references in the cmake files or in the source code to "Resources", so it may be something in either XCode or iOS.

**Comment by rh101** [comment]
Original issue fixed in #1092 for all tests projects, but a copy and paste caused a problem with the cpp-template-default, which is now fixed in #1095 

Will close this once that one is done.
